nex/token/nep5.py

```diff
@@ -40,6 +40,8 @@ def handle_nep51(self, operation, args, token: Token):
         elif operation == 'balanceOf':
             if len(args) == 1:
                 account = args[0]
+                if len(account) != 20:
```

Comment:
Isn't a typical address length 34?

Reply:
It’s 34 in Base58Check, but I believe the addresses in the AVM runtime should be 160-bit == 20 bytes, I thought?